chandra_aca/transform.py

```diff
@@ -79,21 +79,38 @@
                          [0.000273439987106, -0.000000046132060, 0.999999962615285]])
 
 
-def pixels_to_yagzag(row, col, allow_bad=False, flight=False, t_aca=20):
+def pixels_to_yagzag(row, col, allow_bad=False, flight=False, t_aca=20,
+                     pix_zero_loc='edge'):
     """
     Convert ACA row/column positions to ACA y-angle, z-angle.
     It is expected that the row and column input arguments have the same length.
 
+    The ``pix_zero_loc`` parameter controls whether the input pixel values
+    are assumed to have integral values at the pixel center or at the pixel
+    lower/left edge.  The PEA flight coefficients assume lower/left edge, and
+    that is the default.  However, it is generally more convenient when doing
+    centroids and other manipulations to use the center.
+
     :param row: ACA pixel row (single value, list, or 1-d numpy array)
     :param col: ACA pixel column (single value, list, or 1-d numpy array)
     :param allow_bad: boolean switch.  If true, method will not throw errors
                          if the row/col values are nominally off the ACA CCD.
     :param flight: Use flight EEPROM coefficients instead of default ground values.
     :param t_aca: ACA temperature (degC) for use with flight (default=20C)
+    :param pix_zero_loc: row/col coords are integral at 'edge' or 'center'
     :rtype: (yang, zang) each vector of the same length as row/col
     """
     row = np.array(row)
     col = np.array(col)
+
+    if pix_zero_loc == 'center':
+        # Transform row/col values to the convention where the lower/left
+        # corner is at 0.0.  This is needed for the _poly_convert below.
```

Comment:
I got a little confused by the comment on first reading.  Maybe say "Transform row/col values from 'center' convention to 'edge' convention needed for use in _poly_convert below".